### PR TITLE
Make main menu references consistent

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   |  |
| Versions      | 4.5+ |
| Edition       | all |

Make main menu references consistent

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale

### CODE REVIEW

**Review by adamwojs — APPROVED** [review]

File: docs/content_management/forms/create_form_attribute.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Create a `src/FormBuilder/FormSubmission/Converter/RichtextDescriptionFieldSubmi
 ```
 
 Now you can go to Back Office and build a new form.
-In the left menu select **Forms**, click **Create content** and select **Form**.
+In main menu, click **Content** and **Forms**, then click **Create content** and select **Form**.
```
Comment (julitafalcondusza) [inline docs/content_management/forms/create_form_attribute.md:117]:
```suggestion
In the main menu, click **Content** -> **Forms** -> **Create content**, and select **Form**.
```

File: docs/content_management/pages/page_builder_guide.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ By editing pages you can customize the layout and content of your website.
 
 To create a new page:
 
-A\. In the left panel, go to **Content**.
+A\. In main menu, go to **Content**.
```
Comment (julitafalcondusza) [inline docs/content_management/pages/page_builder_guide.md:18]:
```suggestion
A\. In the main menu, go to **Content**.
```

File: docs/customer_management/customer_portal.md [hunk 3]
```diff
@@ -92,9 +92,9 @@ All roles and policies associated with them can be fully customized to fit your
 
 Members can be invited to the organization from:
 
-- the Back Office, go to **Customers** -> **Companies** -> **Select your company** -> **Invitations** -> **Invite member**
+- the Back Office: go to **Customers** -> **Companies** -> **Select your company** -> **Invitations** -> **Invite member**
```
Comment (julitafalcondusza) [inline docs/customer_management/customer_portal.md:95]: We are using comma in such sentences (there is even example below "n the Back Office, go to **Admin** (...)), so I would keep it as it was.
```suggestion
- the Back Office, go to **Customers** -> **Companies** -> **Select your company** -> **Invitations** -> **Invite member**
```

File: docs/customer_management/customer_portal.md [hunk 4]
```diff
@@ -92,9 +92,9 @@ All roles and policies associated with them can be fully customized to fit your
...
 
-- the Customer Portal, go to your company admin profile, select **Members** -> **Invite members**
+- the Customer Portal: go to your company admin profile, select **Members** -> **Invite members**
```
Comment (julitafalcondusza) [inline docs/customer_management/customer_portal.md:97]: Same here:
```suggestion
- the Customer Portal, go to your company admin profile, select **Members** -> **Invite members**
```

File: docs/tutorials/beginner_tutorial/2_create_the_content_model.md [hunk 5]
```diff
@@ -33,7 +33,7 @@ Go to the admin interface (`<yourdomain>/admin`) and log in with the default use
 
 In the upper-right corner, click the avatar icon to unfold the drop-down menu and disable the [Focus mode]([[= user_doc =]]/getting_started/#focus-mode).
 
-Go to the left side menu and choose **Content** -> **Content Types**.
+Go to main menu, go to **Content** -> **Content Types**.
```
Comment (julitafalcondusza) [inline docs/tutorials/beginner_tutorial/2_create_the_content_model.md:36]:
```suggestion
In the main menu, go to **Content** -> **Content Types**.
```

File: docs/tutorials/generic_field_type/5_add_a_field.md [hunk 6]
```diff
@@ -9,7 +9,7 @@ Go to the admin interface (`<yourdomain>/admin`) and log in with the default use
 
 ## Add new Content Type
 
-In the Back Office, in the left menu navigate to the **Content Types** page.
+In the Back Office, the main menu, go to the **Content Types** page.
```
Comment (julitafalcondusza) [inline docs/tutorials/generic_field_type/5_add_a_field.md:12]:
```suggestion
In the Back Office, in the main menu, go to the **Content Types** page.
```

**Review by julitafalcondusza — CHANGES_REQUESTED** [review]
Please see my comments.

File: docs/customer_management/customer_portal.md (hunk 3 above)
Comment (dabrt) [reply docs/customer_management/customer_portal.md:95]: "yes but" we have a preceding part of the sentence that goes "Members can be invited to the organization from..." and "the Back Office:" with a colon completes it nicely. WDYT @julitafalcondusza @juskora ?

Comment (julitafalcondusza) [reply docs/customer_management/customer_portal.md:95]: I would update the sentence: "Members can be invited to the organization from:" -> "Members can be invited to the organization in the following ways:". Then, "**in** the Back Office, (...)".

File: docs/tutorials/beginner_tutorial/2_create_the_content_model.md [hunk 7]
```diff
@@ -68,12 +68,12 @@ Confirm the creation of the Content Type by clicking **Save and close**.
...
     For this tutorial, swap it with its child, a Folder called "Ibexa Platform".
     
-    To do this, go to the left side menu and choose **Content structure** -> **Ibexa Digital Experience Platform**, select the **Locations** tab
+    To do this, in the main menu go to **Content** -> **Content structure** -> **Ibexa Digital Experience Platform**, select the **Locations** tab
     and in the **Swap Locations** section navigate to "Ibexa Platform".
     
     You can learn how to work with Pages in [another tutorial](page_and_form_tutorial.md).
 
-Go back to the content by selecting **Content structure** in the left menu. 
+Go back to the content by selecting **Content structure** in the main menu.
```
Comment (adriendupuis) [inline docs/tutorials/beginner_tutorial/2_create_the_content_model.md:76]:
```suggestion
Go back to the content by selecting **Content structure** in the main menu.
```

**Review by adriendupuis — APPROVED** [review]

**Pull request merged**